ci_tutorial/products_pipeline/multi_product.rst

```diff
@@ -0,0 +1,131 @@
+                            },
+                            {
+                                "package_id": "cbeb3ac76e3d890c630dae5c068bc178e538b090",
+                                "filenames": ["game_debug"],
+                                "build_args": "--requires=engine/1.0 --build=engine/1.0",
+                                
+                            }
+                        ]
+                    ]
+                }
+            ],
+            [
+                {
+                    "ref": "game/1.0#1715574045610faa2705017c71d0000e",
+                    "packages": [
+                        [
+                            {
+                                "package_id": "bac7cd2fe1592075ddc715563984bbe000059d4c",
+                                "filenames": ["game_release"],
+                                "build_args": "--requires=game/1.0 --build=game/1.0",
+                            },
+                            {
+                                "package_id": "01fbc27d2c156886244dafd0804eef1fff13440b",
+                                "filenames": ["game_debug"],
+                                "build_args": "--requires=game/1.0 --build=game/1.0",
+                            }
+                        ]
+                    ]
+                }
+            ]
+        ]
+    }
+
+
+This build order summarizes the necessary builds. First it is necessary to build all different binaries for ``engine/1.0``. This recipe contains 2 different binaries, one for Release and the other for Debug. These binaries belong to the same element in the ``packages`` list, which means they do not depend on each other and can be built in parallel. Each binary tracks its own original build-order file with ``"filenames": ["game_release"],`` so it is possible to deduce the necessary profiles to apply to it.
+
+Then, after all binaries of ``engine/1.0`` have been built, it is possible to proceed to build the different binaries for ``game/1.0``. It also contains 2 different binaries for its debug and release configurations, which can be built in parallel.
+
+In practice, this would mean something like:
+
+.. code-block:: bash
+
+    # This 2 could be executed in parallel 
+    # (in different machines, or different Conan caches)
+    $ conan install --requires=engine/1.0 --build=engine/1.0
+    $ conan install --requires=engine/1.0 --build=engine/1.0 -s build_type=Debug
+
+    # Once engine/1.0 builds finish, it is possible
+    # to build these 2 binaries in parallel (in different machines or caches)
+    $ conan install --requires=game/1.0 --build=game/1.0
+    $ conan install --requires=game/1.0 --build=game/1.0 -s build_type=Debug
+
+In this section we have still omitted some important implementation details that will follow in next sections. The goal was to focus on the ``conan graph build-order-merge`` command and how different products and configurations can be merged in a single "build-order".
```

Comment:
> some important implementation details that will follow in next sections

I'd add some more info, like a "next sections such as..." to give a bit por a peek into the whole picture

Reply:
Done